api/src/accounts.rs

```diff
@@ -39,12 +39,12 @@ pub fn get_account(context: Context) -> BoxedFilter<(impl Reply,)> {
 }
 
 // GET /accounts/<address>/blob
-pub fn get_account_state_blob(context: Context) -> BoxedFilter<(impl Reply,)> {
-    warp::path!("accounts" / AddressParam / "blob")
+pub fn get_account_state(context: Context) -> BoxedFilter<(impl Reply,)> {
```

Comment:
I wish we can  delete this altogether. It's just resources + modules.

Reply:
Good point, deleted this API altogether.